<launch>
  <node pkg="mono_vo_ros" type="mono_vo_node" name="mono_vo_node"  output="screen">
    <param name="max_frame" value="4000" />
    <param name="min_num_pts" value="4000" />
<!--     <param name="fn_kitti" value="/media/shin/T7 Touch/rosbag/KITTI" /> -->
    <param name="fn_kitti" value="/home/shin/dataset/benchmark/KITTI" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mono_vo_ros)/rviz/mono_vo.rviz" output="screen" />
</launch>
